tests/integration/records/single_db/numeric_expectations.py

```diff
@@ -180,8 +180,8 @@
         'uint64': 'NUMERIC(20, 0)',
         'float16': 'REAL',
         'float32': 'REAL',
-        'float64': 'DOUBLE PRECISION',
-        'float128': 'DOUBLE PRECISION',  # Postgres doesn't support >float64
+        'float64': 'DOUBLE_PRECISION',
```

Comment:
What's the purpose of this change?

I can't say it's right or wrong but it's the only change that doesn't "fit" with the others in this PR.

Reply:
Yeah, this is a result of downgrading sqlalchemy to 1.4, where i guess it returns a sligthly different string here